### ISSUE TITLE
NOT able to run it.

### ISSUE THREAD

**Issue by costlife** [openai/openai-quickstart-node#63]
### Describe the bug

I just download the demo, switch the apiKey to my own, and run it.
When I click btn "generate names", the info below shows in my terminal.

400 <html>
<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>
<body>
<center><h1>400 Bad Request</h1></center>
<center>The plain HTTP request was sent to HTTPS port</center>
<hr><center>nginx</center>
</body>
</html>

<img width="485" alt="image" src="https://user-images.githubusercontent.com/10192794/222868293-0bafab5a-0e01-4b15-823f-2128b470206b.png">
<img width="624" alt="image" src="https://user-images.githubusercontent.com/10192794/222868314-9fbf3fa2-141c-4ca3-a0f8-669b24cdfeb7.png">

### To Reproduce

1. gh repo clone openai/openai-quickstart-node 
2. switch the apiKey.
3. npm install & npm run dev
4. input and click the btn.

**Comment by weedsboy** [comment]
same problem

**Comment by TD-winter** [comment]
same problem

**Comment by zhoyou** [comment]
same problem

**Comment by Mystist** [comment]
The server behind the api.openai.com has been blocked from CN, thus lead you to connect it via a proxy.

However, the proxy server will usually be a http server, which did not match the Nginx settings of the api server.

To resolve this, you may need the (HTTP_tunnel)[https://en.wikipedia.org/wiki/HTTP_tunnel#HTTP_CONNECT_Tunneling] technology to create a network link between.

Checkout https://github.com/TooTallNate/node-https-proxy-agent or https://github.com/koichik/node-tunnel for a npm package using this technology.

**Comment by H000CH13** [comment]
Hello. I am completely new to GitHub, Programming, and everything else, but eager to learn. I am glad I found this because I have been having a heck of a time with a similar issue. 

Mine gives me this when I click "generate names" 

![image](https://user-images.githubusercontent.com/130265935/231642328-d43247de-7381-4daa-afcb-2f7669907143.png)

Bear with me as I try to explain what is confusing to me about the tutorial. 

![image](https://user-images.githubusercontent.com/130265935/231642633-aa9711ec-8113-4a56-8db6-acd5e7758046.png)

Here is the part that I dont understand. 'cp .env.example .env' 

When I run that command in GitBash, it looks like this after I hit enter. 

![image](https://user-images.githubusercontent.com/130265935/231642843-7a6b4f12-6b58-4787-b3d0-575fd02421fd.png)

is the file already copied after that command? I downloaded the .zip for the repository also, and created a new .env file following the direction with my API key. was this correct? I feel like I am just missing something fundametally wrong here. 

Thanks. 

-HOOOCH13
